projects/fsastorefrontlib/src/cms-components/assets-table/assets-table.component.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  OnInit,
+} from '@angular/core';
+
+@Component({
+  selector: 'cx-fs-assets-table',
+  templateUrl: './assets-table.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AssetsTableComponent implements OnInit {
```

Comment:
Put some JS doc above this component that explains the purpose of it, like Spartacus did for their TableComponent

Reply:
Added